doc/fairness.ipynb

```diff
@@ -274,7 +274,7 @@
     "\n",
     "The `InformationFilter` uses a variant of the [gram smidt process](https://en.wikipedia.org/wiki/Gram%E2%80%93Schmidt_process) to filter information out of the dataset. We can make it visual in two dimensions;\n",
     "\n",
-    "![proj_img](_static/projections.png)\n",
+    "![proj«img](_static/projections.png)\n",
```

Comment:
interesting. any context on why this character causes the fix?

Reply:
I think the character you're seeing is githubs' way of indicating a character removal. I only removed the underscore from the alt text